### PR TITLE
feat: Added hard-coded ERC20 tokens (such as WASTR that is not supported by cBridge)

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* Added hard-coded ERC20 tokens (such as WASTR that is not supported by cBridge)
* Added the token's explorer link to the EVM assets list.

**Check list**
- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes

**This pull request makes the following changes:**

**Adds**
image

### CODE REVIEW

File: src/components/assets/Erc20Currency.vue [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+              {{ $t('assets.transfer') }}
+            </button>
+            <div v-if="token.isWrappedToken">
+              <a :href="token.wrapUrl" target="_blank" rel="noopener noreferrer">
+                <button class="btn btn--sm">{{ $t('assets.wrap') }}</button>
+              </a>
+            </div>
+            <a :href="explorerLink" target="_blank" rel="noopener noreferrer" class="screen--xl">
+              <button class="btn btn--sm">
+                {{ $t('assets.explorer') }}
```
Comment (hoonsubin) [inline src/components/assets/Erc20Currency.vue:45]: How about an icon button like the one on the account's explorer view?

Comment (Kahonnohak) [reply src/components/assets/Erc20Currency.vue:45]: an icon makes sense to me. It's nice to have more contents:) 

Love the wrap too!  I understand we won't have this in mobile. In the future I'd add three dot icon in mobile view for those extra contents.

Comment (impelcrypto) [reply src/components/assets/Erc20Currency.vue]: Fixed!

**Review by bobo-k2 — APPROVED** [review]
LGTM

**Review by hoonsubin — APPROVED** [review]
lgtm

**Pull request merged**